### ISSUE TITLE
Use dependencies in the INSTALL_PREFIX by default

### ISSUE THREAD

**Issue by lpetre-ulb** [cms-gem-daq-project/gembuild#5]
Title says it all. If the required dependencies exist the the `INSTALL_PREFIX`, they should be found and use automatically by default. This applies to `xhal`, `reedmuller-c` and possibly other GEM libraries.

**Comment by jsturdy** [comment]
I would oppose this on the grounds that `INSTALL_PREFIX` is designed and intended to be "package local", i.e., not telling the build system about other packages (which is not to say that other packages *may* live in there)

**Comment by lpetre-ulb** [comment]
I agree that _this specific proposal_ may not be the best one, for multiple reasons. However, I'm convinced that easy installation in local installation must not be overlooked. It is essential for the easiness of development.

Another solution would be to have a `ADDITIONAL_SEARCH_DIR` which would implement the previously described functionality.
